raiden/api/rest.py

```diff
@@ -46,20 +47,30 @@ class APIServer(object):
         'hexaddress': HexAddressConverter
     }
 
-    # default resource classes will be added to the routes on initialisation and will be exposed
-    # once the RestfulAPI is running
+    # default resource classes will be added to the routes on initialisation
+    # and will be exposed once the RestfulAPI is running
     _default_resource_classes = [
         ChannelsResource,
-        ChannelsByPartner,
-        ChannelsResourceByChannelAddress
+        ChannelsResourceByChannelAddress,
+        TokensResource,
+        PartnersResourceByTokenAddress
     ]
 
     def __init__(self, rest_api):
         self.rest_api = rest_api
-        self.flask_api_middleware = Api(app)
+        self.blueprint = create_blueprint()
+        if self.rest_api.version == 1:
+            self.flask_api_middleware = Api(
+                self.blueprint,
+                prefix="/api/1"
+            )
+        else:
+            raise ValueError('Invalid api version: {}'.format(self.rest_api.version))
 
+        self.flask_app = Flask(__name__)
```

Comment:
For security reasons we may not allow requests from non-local IP addresses [possible solution](http://stackoverflow.com/questions/22251038/how-to-limit-flask-dev-server-to-only-one-visiting-ip-address).

Also, we probably want to configure the ip address and port number that the server is exposing, specially for testing under MacOSX since rebinding on the same port may result in errors.